src/formatters/junitFormatter.ts

```diff
@@ -41,15 +41,20 @@ export class Formatter extends AbstractFormatter {
     };
     /* tslint:enable:object-literal-sort-keys */
 
-    public format(failures: RuleFailure[]): string {
+    public format(failures: RuleFailure[], _fixes?: RuleFailure[], fileNames?: string[]): string {
         let output = '<?xml version="1.0" encoding="utf-8"?><testsuites package="tslint">';
 
+        const failureFileNames: Set<string> = new Set<string>();
```

Comment:
This line could be written as such, then the `if` stmt below could be removed.
```ts
const failureFileNames: Set<string> = new Set([...failures.map(f => f.getFileName())]);
```

Reply:
Good call. I've added that.